### DISCUSSION TITLE
Does it work when run with ts-node with `--transpileOnly` flag?

### DISCUSSION THREAD

**Discussion by capaj** [Hookyns/tst-reflect#71] [Q&A]
I'd like to get reflection when I run my code in ts-node with `--transpileOnly` flag.
Does reflection work without typechecking by any chance?

I work on bigger codebases, where running with typechecks is very slow.

**Comment by Hookyns** [comment]
Hello @capaj, long time no see.

I never tested it, but yes, it seems it does. I've just tried ts-node with `--transpileOnly`.
Try this [example](https://github.com/Hookyns/tst-reflect/tree/main/examples/ts-node-transpile-only).
